lm_eval/tasks/score/NON_GREEDY.md

```diff
@@ -0,0 +1,32 @@
+```
+Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+````
+# How to run evaluation on SCORE Non-Greedy task?
+
+Evaluation for non greedy tasks differs a bit from other score tasks as it is required to pass different seeds as an argument manually. Below you can find the step-by-step guide on how to correctly run the **Score Non-Greedy** evaluation.
+
+To run the evaluation of the Non-Greedy tasks with 5 different seeds you should:
+1. For a given dataset run the evaluation by
+   * specifying the task as `score_non_greedy_robustness_{DATASET_NAME}` (`DATASET_NAME` being either`agieval`, `mmlu_pro` or `math`)
+   * fixing the seed with the run argument `--seed=1`
+   * passing the `--log_samples` argument
+   * specifying an output with `--output_path=SOME_OUTPUT_PATH/seed_1`
+   * if running with vllm it is important to set the seed in the `--model_args` as well buy jsut specifying the `seed` parameter
```

Comment:
"buy jsut" -> "by just"

Reply:
Any caution message here that prediction will take some space? and why file saving is required